.github/workflows/code-lint.yml

```diff
@@ -28,3 +28,20 @@ jobs:
         . ./ci/common.sh
         export_or_prefix
         make lint
+
+  sc-lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.4.0
+        with:
+          submodules: true
+
+      - name: Shellcheck code
+        run: |
+          scversion="latest"
+          wget -O- "https://github.com/koalaman/shellcheck/releases/download/${scversion?}/shellcheck-${scversion?}.linux.x86_64.tar.xz" | tar -xJv
+          cp -av "shellcheck-${scversion}/shellcheck" /usr/local/bin/
+          shellcheck --version
+          shellcheck **/*.sh
```

Comment:
Let's limit the scope. We don't want to lint the code from git submodule.

Reply:
Ok, I got the meaning that checkout repo without submodule, so it won't lint the code from git submodule.

Reply:
> Let's limit the scope. We don't want to lint the code from git submodule.

Done.

Reply:
> shellcheck **/*.sh

? It is still linting all the shell files in this repo, including the one from submodule...